platform/solr/solr-factory/src/main/java/org/codice/solr/factory/impl/ConfigurationFileProxy.java

```diff
@@ -40,14 +39,21 @@
 
     public static final String DEFAULT_SOLR_CONFIG_PARENT_DIR = "etc";
 
-    public static final String SOLR_CONFIG_LOCATION_IN_BUNDLE =
-            new AbsolutePathResolver("solr/conf").getPath();
-
     public static final String DEFAULT_SOLR_DATA_PARENT_DIR =
             new AbsolutePathResolver("data/solr").getPath();
 
     public static final String CATALOG_SOLR_COLLECTION_NAME = "metacard";
 
+    protected static final List<String> SOLR_CONFIG_FILES =
```

Comment:
`private`?

And should we inject this list from the blueprint file instead of hard-coding it here? This would make the unit tests a little simpler as the only thing they would need to verify is that all the files it injects got processed and wouldn't need to worry about the real list.